src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -377,6 +383,7 @@ public void removeNode(String nodeName) {
     /*** Internal API *************************************************************************************************/
     class InternalEndpoint implements CloudioNodeContainer, MqttCallback, Runnable {
         /*** Constants ************************************************************************************************/
+        private static final String UUID			                = "ch.hevs.cloudio.endpoint.uuid";
```

Comment:
`UUID_PROPERTY`

Reply:
@RandomReaper To be consistent, we should use UUID_PROPERTY for the key like all others